plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -78,25 +77,34 @@ public String getFileName() {
   /**
    * Directory containing the generated Docker info JAR.
    */
-  @Parameter(defaultValue = "${project.build.directory}", property = "dockerfile.outputDirectory", required = true)
+  @Parameter(defaultValue = "${project.build.directory}",
+      property = "dockerfile.outputDirectory",
+      required = true)
```

Comment:
I don't think the google style guide says anything explicitly. https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break. How'd you do it?